api/v1alpha1/backstage_types.go

```diff
@@ -19,15 +19,12 @@ import (
 )
 
 const (
-	RuntimeConditionRunning string = "RuntimeRunning"
-	RuntimeConditionSynced  string = "RuntimeSyncedWithConfig"
-	RouteSynced             string = "RouteSynced"
-	LocalDbSynced           string = "LocalDbSynced"
-	SyncOK                  string = "SyncOK"
-	SyncFailed              string = "SyncFailed"
-	Deleted                 string = "Deleted"
-	EnvPostGresImage        string = "RELATED_IMAGE_postgresql"
-	EnvBackstageImage       string = "RELATED_IMAGE_backstage"
+	//RuntimeConditionRunning string = "RuntimeRunning"
+	ConditionSynced   string = "Synced"
+	SyncOK            string = "SyncOK"
+	SyncFailed        string = "SyncFailed"
+	EnvPostGresImage  string = "RELATED_IMAGE_postgresql"
```

Comment:
nit: can we move the `Env*Image` constants into a separate `const` group? I find it more readable to group related declarations..

Reply:
Good idea. Done.